src/Internal/Metadata/Cip25/Cip25String.purs

```diff
@@ -142,5 +142,5 @@ toMetadataString str = case toCip25Strings str of
 fromMetadataString :: TransactionMetadatum -> Maybe String
 fromMetadataString datum = do
   fromCip25Strings <$> (Array.singleton <$> fromMetadata datum) <|> do
-    bytes :: Array ByteArray <- fromMetadata datum
-    hush $ decodeUtf8 $ unwrap $ fold bytes
+    strings :: Array Cip25String <- fromMetadata datum
+    pure $ foldMap (\(Cip25String s) -> s) strings
```

Comment:
We can use the existing function to simplify to:

```purescript
fromMetadataString :: TransactionMetadatum -> Maybe String
fromMetadataString datum = do
  fromCip25Strings <$> (Array.singleton <$> fromMetadata datum) <|>
    fromCip25Strings <$> fromMetadata datum
```

Reply:
I think this can even be simplified to:
```
fromMetadataString :: TransactionMetadatum -> Maybe String
fromMetadataString datum = do
  fromCip25Strings <$> (Array.singleton <$> fromMetadata datum <|> fromMetadata datum)
```